CONTRIBUTING.md

```diff
@@ -10,35 +10,54 @@ Contributing
 Testing
 =======
 
-Running the tests requires [Grunt CLI](https://github.com/gruntjs/grunt-cli). It
-is available via [npm](https://npmjs.org):
+Install the dependencies for the project:
 
 ```
-npm install grunt-cli
+npm install
 ```
 
-Also install the dependencies for the project:
+### Headless (Using PhantomJS)
 
 ```
-npm install
+npm run test:quick
 ```
 
 ### In browser
 
+Any large changes should be tested in old IEs (we support IE 6!), and any other
+browsers you can get your hands on.
+
+#### Sauce Labs automated browser testing
+
+To do automated browser testing with sauce labs first install sauce-connect
+
 ```
-grunt browsertest
+brew cask install sauce-connect
 ```
 
-Any large changes should be tested in old IEs (we support IE 6!), and any other
-browsers you can get your hands on. The easiest way
-to get these is from [modern.ie](https://www.modern.ie/en-gb/virtualization-tools#downloads).
+Then ensure that you have `SAUCE_USERNAME` and `SAUCE_ACCESS_KEY` environment
+variables set.
 
-### Headless (Using PhantomJS)
+Finally run:
+
+```
+karma start --single-run
```

Comment:
If we're going to reference the karma commands directly in the docs, it would be nice to tell people to `npm install -g karma-cli` at some point, maybe when they are installing sauce-connect.